test/list/ListProxyTest.js

```diff
@@ -259,15 +260,41 @@ describe("List Proxy", function () {
         })
     });
 
+    it("listens for added and removed entry", function (done) {
+        this.timeout(5000);
+        var listener = {
+            itemAdded: function (itemEvent) {
```

Comment:
even if this method is never called, the test would pass.

Reply:
I will add a boolean check for it.